### PR TITLE
Td 1439/gh output for diagnostician

### PR DESCRIPTION
**Pull request by ermirizio**

### CODE REVIEW

File: policy/templates/releng/ci/goreleaser/goreleaser.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ checksum:
   disable: true
 
 release:
-  disable: true
+  disable: tr221
```
Comment (alephnull) [inline policy/templates/releng/ci/goreleaser/goreleaser.yml:31]: ?

File: git/diff.go [hunk 2]
```diff
@@ -11,14 +11,14 @@ import (
...
-	cmd := exec.Command("git", "diff", "-G", "-w", "--ignore-all-space", "(^[^#])", "HEAD")
+	cmd := exec.Command("git", "diff", "-G", "(^[^#])", "-w", "--ignore-all-space", "HEAD")
 	cmd.Dir = dir
 	cmd.Stdout = &out
 	err := cmd.Run()
 	if err != nil {
 		return "", fmt.Errorf("diff in %s: %w", dir, err)
 	}
-	log.Trace().Bytes("output", out.Bytes()).Str("dir", dir).Msg("git diff")
+	log.Info().Bytes("output", out.Bytes()).Str("dir", dir).Msg("git diff")
```
Comment (alephnull) [inline git/diff.go:21]: This is not very readable. Also, it doesn't make sense to use the logger for output. 
What I suggest is to add a flag to the diff subcommand which will also print the diff string, nicely formatted.

**Review by alephnull — CHANGES_REQUESTED** [review]
Is this ready for review? It looks like it is in the middle of development.

File: policy/templates/releng/ci/goreleaser/goreleaser.yml (hunk 1 above)
Comment (ermirizio) [reply policy/templates/releng/ci/goreleaser/goreleaser.yml:31]: Hi Alok, yes is actually in development I will turn this into draft until is ready for review

File: git/diff.go (hunk 2 above)
Comment (ermirizio) [reply git/diff.go:21]: Thanks for the input, I will change the output and add a pretty print as well. I was more dealing with GH and annotations which seem they doesn't support multiline string 👎

Comment (alephnull) [reply git/diff.go:21]: Ah, I see. Perhaps just the single line summary that is printed now might be enough?

**Review by alephnull — APPROVED** [review]

**Pull request merged**